code/datums/elements/volatile_gas_storage.dm

```diff
@@ -0,0 +1,46 @@
+/datum/element/volatile_gas_storage
+	element_flags = ELEMENT_BESPOKE
+	id_arg_index = 2
+
+	var/minimum_explosive_pressure
+	var/max_explosive_pressure
+	var/max_explosive_force
+
+/datum/element/volatile_gas_storage/Attach(datum/target, minimum_explosive_pressure=5000, max_explosive_pressure=100000, max_explosive_force=9)
+	. = ..()
+	if(istype(target, /obj/machinery/portable_atmospherics))
+		RegisterSignal(target, COMSIG_OBJ_BREAK, .proc/PortableBreak)
+	else if(istype(target, /obj/machinery/atmospherics/components))
+		RegisterSignal(target, COMSIG_OBJ_BREAK, .proc/AtmosComponentBreak)
+	else
```

Comment:
Why is there a case for this? is it something you plan on using downline?

Reply:
I'm not sure what you mean, this is just the else for handling when the target isn't any of the supported types

Reply:
I'm asking why you're supporting components here, since I can't seem to find any use of this on them in this pr, and they seem like an odd use case

Reply:
To reiterate what we talked about in discord: They're the real reason I made this change so I can use this for stationary tanks later. There's no harm in adding the support here so it doesn't need to be part of the larger pr.